aptos-move/aptos-release-builder/src/components/feature_flags.rs

```diff
@@ -285,7 +285,9 @@ impl From<FeatureFlag> for AptosFeatureFlag {
             FeatureFlag::ReconfigureWithDkg => AptosFeatureFlag::_DEPRECATED_RECONFIGURE_WITH_DKG,
             FeatureFlag::KeylessAccounts => AptosFeatureFlag::KEYLESS_ACCOUNTS,
             FeatureFlag::KeylessButZklessAccounts => AptosFeatureFlag::KEYLESS_BUT_ZKLESS_ACCOUNTS,
-            FeatureFlag::RemoveDetailedError => AptosFeatureFlag::REMOVE_DETAILED_ERROR_FROM_HASH,
+            FeatureFlag::RemoveDetailedError => {
```

Comment:
this can be renamed to `_deprecated_XX` as well?

Reply:
thought of this. But see others like this https://github.com/aptos-labs/aptos-core/blob/deprecate_related_changes/aptos-move/aptos-release-builder/src/components/feature_flags.rs#L219 not adding _deprecated. want to keep it consistent.